components/Button/Button.tsx

```diff
@@ -38,7 +38,7 @@ const buttonVariants = cva(
 export interface ButtonProps
   extends React.ButtonHTMLAttributes<HTMLButtonElement>,
     VariantProps<typeof buttonVariants> {
-  label?: string;
+  label?: React.ReactNode;
```

Comment:
Are we sure this is always going to be a ReactNode?

Reply:
Hmm I guess not. I believe I should have just left it as a string, no?